code/modules/power/engines/singularity/collector.dm

```diff
@@ -80,6 +80,11 @@ GLOBAL_LIST_EMPTY(rad_collectors)
 		if(loaded_tank)
 			to_chat(user, "<span class='notice'>Remove the plasma tank first.</span>")
 			return TRUE
+		var/turf/T = get_turf(src)
+		for(var/obj/machinery/power/rad_collector/can_wrench in T.contents)
+			if(istype(can_wrench, /obj/machinery/power/rad_collector) && can_wrench.anchored && !anchored)
+				to_chat(user, "<span class='notice'>You can't wrench down [src] here!</span>")
+				return
```

Comment:
No need to check here, since you already typecasted by checking in the for loop
```suggestion
			if(can_wrench.anchored && !anchored)
				to_chat(user, "<span class='notice'>You can't wrench down [src] here!</span>")
				return
```

Reply:
guh I really need to start paying attention more